api/all/src/main/java/io/opentelemetry/api/internal/InternalAttributeKeyImpl.java

```diff
@@ -52,13 +58,17 @@ public String getKey() {
     return key;
   }
 
+  public byte[] getKeyUtf8() {
```

Comment:
Could you get the best of both worlds by having this extra method, but instead of preencoding it at construction time, lazily encoding it when `getKeyUtf8()` is called, and memoizing the result?

Reply:
If it's good enough for Java hashcode, it's good enough for me thanks for the suggestion